packages/yoga/src/AutoComplete/web/AutoComplete.test.jsx

```diff
@@ -65,6 +65,25 @@ describe('<AutoComplete />', () => {
       );
     });
 
+    it('should not filter the options', () => {
+      const { container, getByDisplayValue } = render(
+        <ThemeProvider>
+          <AutoComplete
+            value="secon"
+            options={['first', 'second', 'third']}
+            shouldfilterOptions={false}
```

Comment:
```suggestion
            shouldFilterOptions={false}
```
And in the component as well